api/MediaRecorder.json

```diff
@@ -164,14 +164,7 @@
             },
             "chrome": {
               "version_added": "47",
-              "notes": "Currently only video is supported, not audio.",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "Experimental Web Platform features",
-                  "value_to_set": "Enabled"
-                }
-              ]
+              "notes": "Prior to Chrome 49, only video is supported, not audio."
```

Comment:
This information applies to Android and Android Webview as well. Please change. https://www.chromestatus.com/features/5929649028726784

Reply:
Thanks! Updated for Android and Android Webview, also added data for Opera desktop/Android.